### PR TITLE
improve docstrings

### PR DESCRIPTION
**Pull request by bendichter**

### CODE REVIEW

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
Aha, the read the docs CI indeed found a mistake

These functions were not rendered in the API docs due to not having specified docstrings, as such the inter references with the best practices were not getting tested

```
/nwbinspector/checks/ophys.py:docstring of nwbinspector.checks.ophys.check_plane_segmentation_image_mask_shape_against_ref_images:3:

WARNING: undefined label: 'best_practice_plane_segmentation_image_mask_shape_against_ref_images'
```

Just need to fix that and it's good to go

**Comment by bendichter** [comment]
@CodyCBakerPhD I'm not clear on how to fix that. Can you do it?

File: docs/best_practices/optical_physiology.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Plane Segmentation
+------------------
+
+
+.. _best_practice_plane_segmentation_image_mask_shape_against_ref_images:
+
+Image Shape Consistency
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``image_mask`` column of a ``PlaneSegmentation``, if specified, should have the same image shape as each item in the ``reference_images``.
```
Comment (CodyCBakerPhD) [inline docs/best_practices/optical_physiology.rst]: Feel free to add anything else here

File: src/nwbinspector/checks/behavior.py [hunk 2]
```diff
@@ -7,7 +7,11 @@
 
 @register_check(importance=Importance.CRITICAL, neurodata_type=SpatialSeries)
 def check_spatial_series_dims(spatial_series: SpatialSeries):
-    """Check if a SpatialSeries has the correct dimensions."""
+    """
+    Check if a SpatialSeries has the correct dimensions.
+
+    Best Practice: :ref:`_best_practice_unit_of_measurement`
```
Comment (CodyCBakerPhD) [inline src/nwbinspector/checks/behavior.py]: There could be a specific section to discuss these particular checks, but for now just linking to the common one

**Comment by CodyCBakerPhD** [comment]
> @CodyCBakerPhD I'm not clear on how to fix that. Can you do it?

What it really means is that there is no Best Practice by that name, which can mean there is no Best Practice section for it or the section reference label is named something different

In this case there was no BP section at all for ophys

File: src/nwbinspector/checks/behavior.py (hunk 2 above)
Comment (bendichter) [reply src/nwbinspector/checks/behavior.py]: I'll write this section now

**Comment by CodyCBakerPhD** [comment]
Docs look great now

GitHub Actions has been intermittent today, as well as the codecov upload (only item failing aside from the ongoing discussion with the DANDI dev branch)

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**